metadata/groups/static-website-azure.yaml

```diff
@@ -0,0 +1,12 @@
+name: Azure Static Website
+kind: architecture
+parent: static-website
+slug: azure
+clouds:
+  - azure
```

Comment:
I'm guessing some time down the road it would be possible that there will be groupings that are associated with multiple clouds?

Reply:
I was thinking that might be possible, yeah (though certainly not common), so just figured I'd leave room for it.